scripts/pi-hole/php/get.php

```diff
@@ -11,57 +11,100 @@
 
 $listtype = $_GET['list'];
 
-$basedir = "/etc/pihole/";
+require_once("func.php");
 
-require_once "func.php";
-
-switch ($listtype) {
-    case "white":
-        $list = array(getListContent("whitelist.txt"));
-        break;
-
-    case "black":
-        $exact = getListContent("blacklist.txt");
-        $regex = getListContent("regex.list");
-        $list = array($exact, $regex);
-        break;
-
-    default:
-        die("Invalid list parameter");
-        break;
+// Get possible non-standard location of FTL's database
+$FTLsettings = parse_ini_file("/etc/pihole/pihole-FTL.conf");
+if(isset($FTLsettings["GRAVITYDB"]))
+{
+	$GRAVITYDB = $FTLsettings["GRAVITYDB"];
+}
+else
+{
+	$GRAVITYDB = "/etc/pihole/gravity.db";
 }
 
+function SQLite3_connect($dbfile, $trytoreconnect)
```

Comment:
This function is duplicated here from `api_db.php`, can it be moved to a common file and imported?